dali/test/python/test_operator_readers_numpy.py

```diff
@@ -108,7 +108,7 @@ def check_batch(test_data_root, batch_size, num_samples, device, arr_np_all, fil
                                    batch_size = batch_size,
                                    num_threads = num_threads,
                                    device_id = 0,
-                                   cache_header_information = False)
+                                   cache_header_information = cache_header_information)
```

Comment:
Did fixing the test before fixing the reader make the test fail?

Reply:
Yes, it did.